x-pack/plugins/drilldowns/public/components/form_drilldown_wizard/form_drilldown_wizard.scss

```diff
@@ -0,0 +1,4 @@
+.drdFormDrilldownWizard__formRow .euiFormRow__label {
```

Comment:
We can remove this.  I know this is in the figma file and apologize for the confusion on this.  But I think for now, it'll be best to just keep this standard EUI label sizes

Reply:
Removed https://github.com/elastic/kibana/pull/59632/commits/2b162e829e9fe2a72384dbb057958319fca1772e